tests/integration/test_ping.py

```diff
@@ -0,0 +1,56 @@
+import os
+import pytest
+
+from tests.helpers import IntegrationTestCaseBase
+
+
+@pytest.mark.integration
+class TestPing(IntegrationTestCaseBase):
+    def setup_method(self):
+        super(TestPing, self).setup_method()
+        self.rebirthdb_host=os.getenv('REBIRTHDB_HOST')
+
+    def teardown_method(self):
+        with self.r.connect(host=self.rebirthdb_host) as conn:
+            curr = self.r.db("rethinkdb").table("users").filter(
+                self.r.row["id"].ne("admin")
+            ).delete().run(conn)
+            assert list(curr)
+        super(TestPing, self).teardown_method()
+
+    def test_bad_password(self):
+        with pytest.raises(self.r.ReqlAuthError):
+            self.r.connect(password="0xDEADBEEF", host=self.rebirthdb_host)
```

Comment:
👍 for using `DEADBEEF`, I learned something new today. What do you think about extracting it to a variable? It is used several times.

Reply:
Will do.